websocket_push_session.go

```diff
@@ -211,6 +211,17 @@ func (s *wsPushSession) sendWSError(ee elemental.Error) {
 	}
 }
 
+func (s *wsPushSession) currentPushConfigID() (string, bool) {
+	s.currentPushConfigLock.RLock()
+	defer s.currentPushConfigLock.RUnlock()
+
+	if s.pushConfig == nil || s.pushConfig.ID == "" {
+		return "", false
```

Comment:
what's the use of the returned boolean here? It looks pretty useless

Reply:
It's to indicate that a push configuration has been provided by the client. Otherwise, you would be doing a check on the empty string which is the default value for an uninitialized string.